src/igvfd/schemas/workflow.json

```diff
@@ -97,6 +97,17 @@
                 "appscript": "/help/chip-seq-standards",
                 "igvf_utils": "/help/chip-seq-standards"
             }
+        },
+        "workflow_version": {
+            "title": "Workflow version",
```

Comment:
Small capitalization request, both words should be capitalized (Workflow Version)